api/working-with-extensions/publishing-extension.md

```diff
@@ -62,7 +62,7 @@ Visual Studio Code uses [Azure DevOps](https://azure.microsoft.com/services/devo
 
 `vsce` can only publish extensions using [Personal Access Tokens](https://learn.microsoft.com/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate). You need to create at least one in order to publish an extension.
 
-### Get a Personal Access Token
+### Getting a Personal Access Token
```

Comment:
Our convention is to avoid gerunds in headings (though we're not very strict about enforcing it).
This will also break anyone external bookmarks to that section so we prefer not to change H2 heading.